Interpreting-GitHubActions-Results.md

```diff
@@ -0,0 +1,13 @@
+# Interpreting GitHub Actions Results
+
+## Detailed Explanation
```

Comment:
Add some info on skipped checks and how to view the logs

Reply:
I have added the info.